Source/Core/DolphinQt/Config/Graphics/BalloonTip.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QIcon>
+#include <QPixmap>
+#include <QWidget>
+
+class BalloonTip : public QWidget
+{
+  Q_OBJECT
+
+  struct PrivateTag
+  {
+  };
+
+public:
+  static void ShowBalloon(const QIcon& icon, const QString& title, const QString& msg,
+                          const QPoint& pos, QWidget* parent, bool show_arrow = true);
```

Comment:
Consider replacing the boolean with an enum class (`ShowArrow` with members `Yes` and `No`) to improve readability at call sites.

If somebody passes false as the last argument to ShowBalloon, there is no way of figuring out what that boolean does without checking the function prototype and remembering what the last parameter is called.